README.md

```diff
@@ -1,16 +1,65 @@
 # test-operator
 This is a test operator that automates testing of an OpenStack deployment
-running on a Kubernetes cluster. The operator runs
-[Tempest](https://opendev.org/openstack/tempest) tests.
+running on a OpenShift cluster. The operator supports execution of the following
+tests:
+
+- [Tempest](https://opendev.org/openstack/tempest)
+
+- [Tobiko](https://tobiko.readthedocs.io/en/0.4.7/index.html)
+
+- [Horizon tests](https://review.opendev.org/openstack/horizon)
```

Comment:
I don't know if it's just me, but the Horizon link does not take me anywhere.

Reply:
Thanks!